app/controllers/podcasts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PodcastsController < ApplicationController
+  before_action :set_podcast, only: [:show, :edit, :update, :destroy]
+
+  # GET /podcasts
+  # GET /podcasts.json
+  def index
+    @podcasts = Podcast.all
+  end
+
+  # GET /podcasts/1
+  # GET /podcasts/1.json
+  def show
+  end
+
+  # GET /podcasts/new
+  def new
+    @podcast = Podcast.new
+  end
+
+  # GET /podcasts/1/edit
+  def edit
+  end
+
+  # POST /podcasts
+  # POST /podcasts.json
+  def create
+    @podcast = Podcast.new(podcast_params)
+
+    respond_to do |format|
+      if @podcast.save
+        format.html { redirect_to @podcast, notice: 'Podcast was successfully created.' }
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.